api/routers/main.js

```diff
@@ -1,7 +1,8 @@
 const router = require('express').Router();
+const passport = require('../policies/passport');
 const MainController = require('../controllers/main');
 
-router.get('/', MainController.index);
-router.get('/sites(/*)?', MainController.index);
+router.get('/', passport, MainController.index);
+router.get('/sites(/*)?', passport, MainController.index);
```

Comment:
No worries. I actually think `req.session.authenticated` is available through the session middleware and not passport so we may still be able to pull it out.